src/main/java/code401challenges/tree/BreadthSearch.java

```diff
@@ -0,0 +1,25 @@
+package code401challenges.tree;
+
+
+import java.util.LinkedList;
+import java.util.Queue;
+
+public class BreadthSearch {

```

Comment:
Instead of making a whole class for this, you can just add a method to your Tree class.